doc/changelog.asciidoc

```diff
@@ -19,6 +19,12 @@ breaking changes (such as renamed commands) can happen in minor releases.
 v3.3.0 (unreleased)
 -------------------
 
+Added
+~~~~~
+
+- A setting for disabling the Google Hangouts extension,
+  built into Chromium/QtWebEngine.
```

Comment:
nit: maybe mention the setting name here, eg "added the `qt.workarounds.disable_hangouts_extension` setting ..."